lunatrace/bsl/frontend/src/pages/project/build/finding/VulnerablePackageList.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import React, {useState} from 'react';
+import { Container, Row } from 'react-bootstrap';
+import { useNavigate } from 'react-router-dom';
+import { VulnerablePackageItem} from "./VulnerablePackageItem";
+import {Finding} from "./types";
+interface FindingListProps {
+    findings: Finding[];
+}
+import {severityOrder, VulnerablePackage} from "./types";
+
+function groupByPackage(findings:Finding[]): VulnerablePackage[] {
```

Comment:
`findings:Finding[]` looks like the linter hasn't run?

Reply:
AHAH
its not running here because this directory is called "build"
eureka